docs/pages/router/reference/troubleshooting.mdx

```diff
@@ -5,6 +5,16 @@ description: Fixing common issues with Expo Router setup.
 
 import { Terminal } from '~/ui/components/Snippet';
 
+## Missing files / source maps in React Native DevTools
+
+This can happen if your Chrome DevTools has exclusions within its ignore list. To fix the issue
+
+1. Start the React Native DevTools
+2. Open settings by pressing the gear icon
+3. Press "restore defaults and reload"
+4. Open settings again, and click the "ignore list" tab
+5. Remove any exclusions with `node_modules`
```

Comment:
This last step is confusing because if you want devs to uncheck the existing box or add a new *Custom pattern**?

![CleanShot 2024-12-12 at 18 48 11@2x](https://github.com/user-attachments/assets/a4d55f98-1fb6-42de-ba60-cbb2451e0032)